lib/cdo/contact_rollups/v2/pardot.rb

```diff
@@ -176,18 +180,32 @@ def batch_update_remaining_prospects
   # @param [String] api_endpoint
   # @param [Array<Hash>] prospects
   # @param [Boolean] eager_submit if is true, triggers submitting request immediately
+  # @param [String] batch is to create new prospects ('create') or update existing ones ('update')
   # @return [Array<Array>] two arrays, one for all submitted prospects and one for Pardot errors
-  def process_batch(api_endpoint, prospects, eager_submit)
+  def process_batch(api_endpoint, prospects, eager_submit, batch_type)
```

Comment:
How about re-using `api_endpoint` instead of adding a new `batch_type` argument? You can print out the first request hitting each endpoint and ignore the rest.

Reply:
Then `@dry_run_batches_built` array could become something like a `@dry_run_api_endpoints_hit` set.

Reply:
Done!